client/transport/src/ws/mod.rs

```diff
@@ -434,11 +437,11 @@ impl TryFrom<Uri> for Target {
 			Some("ws") => Mode::Plain,
 			#[cfg(feature = "tls")]
 			Some("wss") => Mode::Tls,
-			_ => {
+			invalid_scheme => {
 				#[cfg(feature = "tls")]
-				let err = "URL scheme not supported, expects 'ws' or 'wss'";
+				let err = format!("{:?} not supported, expects 'ws' or 'wss'", invalid_scheme);
 				#[cfg(not(feature = "tls"))]
-				let err = "URL scheme not supported, expects 'ws'";
+				let err = format!("{:?} not supported, expects 'ws' ('wss' requires the tls feature)", invalid_scheme);
```

Comment:
This will print something like `Some("http") not supported, expects 'ws' or 'wss'`; are we happy with that or would it be better to maybe do something like `invalid_scheme.unwrap_or("no scheme")`?

Reply:
Right, I think you suggestion is better.